addon/components/animated-each.js

```diff
@@ -12,16 +12,98 @@ import Sprite from '../-private/sprite';
 import { componentNodes, keyForArray } from '../-private/ember-internals';
 import partition from '../-private/partition';
 
+/**
+  A drop in replacement for `{{#each}}` that animates changes to a list. 
+  ```hbs
+    {{#animated-each items use=transition as |item|}}
+      <div onclick={{action removeItem item}}>
+        {{item}}
+      </div>
+    {{/animated-each}}
+  ```
+  ```js
+  import Component from '@ember/component';
+  import move from 'ember-animated/motions/move';
+  import { fadeOut } from 'ember-animated/motions/opacity';
+
+    export default class extends Component {
```

Comment:
A general idea on getting the indentation right (since it's harder to do in comments): get the demo files exactly how you want them and then do a complete copy/paste from the demo files into the comments, bringing along the indentation.